sdk/typescript/src/otel/tracing.ts

```diff
@@ -8,17 +8,32 @@ import {
 } from '@opentelemetry/sdk-trace-node';
 import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
 
-import Instrumentations from './instrumentation';
-import OpenlitConfig from './config';
+import Instrumentations from '../instrumentation';
+import OpenlitConfig from '../config';
 
 export default class Tracing {
   static traceProvider: NodeTracerProvider;
   static traceExporter: OTLPTraceExporter;
   static async setup(options: SetupTracerOptions) {
     if (options.tracer) return options.tracer;
     try {
+      const consoleSpanExporter = new ConsoleSpanExporter();
+      const url = options.otlpEndpoint + "/v1/traces" || 'http://localhost:4318/v1/traces';
```

Comment:
@GVengelen Can we add a check if the user already has `/v1/traces` in the url endpoint then don't append else append `/v1/traces` ? Also we have https://github.com/openlit/openlit/blob/main/sdk/typescript/src/config.ts where we set config data, it should ideally get updated there as well.